docs/how-to-guides/manage-label-based-access-control-permissions.md

```diff
@@ -0,0 +1,226 @@
+---
+id: manage-label-based-access-control-permissions
+title: How to manage label-based access control permissions (Enterprise)
+sidebar_label: Manage label-based access control permissions
+---
+
+[![Related - Reference Guide](https://img.shields.io/static/v1?label=Related&message=Reference%20Guide&color=yellow&style=for-the-badge)](/reference-guide/security.md)
```

Comment:
Anchor to that chapter in the reference guide...
YOu can create an issue for me to add it once everything is merged